src/server.js

```diff
@@ -212,10 +211,12 @@ class Server {
     if (!isFunction(task)) {
```

Comment:
the caller of this function already takes care of the conversion.
https://github.com/Runnable/ponos/blob/add-max-retry/src/server.js#L190